recipe/meta.yaml

```diff
@@ -297,25 +284,31 @@ outputs:
         - wheel
       run:
         - python
-        - joblib <2,>=1.1
-        - numpy <1.29,>=1.21
-        - scipy <1.13,>=1.5.4
-        - pandas <2.2.0,>=2.0.0
-        - pytorch <2.1,>=2.0
-        - lightning <2.1,>=2.0.0
-        - pytorch-lightning <2.1,>=2.0.0
-        - statsmodels <0.15,>=0.13.0
-        - gluonts <0.15,>=0.14.0
-        - networkx <4,>=3.0
-        - statsforecast <1.5,>=1.4.0
-        - mlforecast <0.10.1,>=0.10.0
-        - utilsforecast <0.0.11,>=0.0.10
-        - tqdm <5,>=4.38
-        - orjson >=3.9,<3.10
-        - tensorboard <3,>=2.9
-        - {{ pin_subpackage(name ~ '.core', exact=True) }}
-        - {{ pin_subpackage(name ~ '.common', exact=True) }}
-        - {{ pin_subpackage(name ~ '.tabular', exact=True) }}
+        - joblib >=1.1,<2
+        - lightning {{ lightning }}
+        - pytorch-lightning {{ lightning }}
+        ## transformers[sentencepiece] ##
+        - transformers {{ transformers }}
+        - sentencepiece {{ sentencepiece }}
+        - protobuf
+        ## transformers[sentencepiece] ##
+        - huggingface_accelerate {{ huggingface_accelerate }}
+        - gluonts ==0.15.1
+        - statsforecast >=1.4.0,<1.5
+        - mlforecast >=0.10.0,<0.10.1
+        - utilsforecast >=0.0.10,<0.0.11
+        - orjson >=3.9,<4
+        - tensorboard >=2.9,<3
+        - {{ pin_subpackage('autogluon.core', exact=True) }}
+        - {{ pin_subpackage('autogluon.common', exact=True) }}
+        - {{ pin_subpackage('autogluon.tabular', exact=True) }}
+        # Extra group "all". autogluon requires autogluon.timeseries[all].
+        - optimum >=1.17,<1.19
+        - onnx
+        - onnxruntime >=1.11.0
+        - datasets >=1.2.1
+        - evaluate
+        - protobuf >=3.20.1
```

Comment:
`protobuf` is specified twice

Reply:
I wanted to leave `protobuf` in the `transformers[sentencepiece]` section above to improve readability/maintainability. I can remove it if it doesn't achieve that.

Reply:
Removed in [cd35cbe](https://github.com/AnacondaRecipes/autogluon-feedstock/pull/2/commits/cd35cbeb9001ce2aa703e0be4cbcb043a7e56a8e).